src/posts/paper/13-q-and-a.ts

```diff
@@ -179,6 +179,54 @@ However, we have retracted this method for the following reasons:
 * Various studies, such as [Baker et al.](https://www.acpjournals.org/doi/10.7326/M20-7145) have shown that, for at least some pockets of healthcare workers, healthcare workers are no more likely to get infected through their work than the average person is to get infected through socializing.
 * The above Washington State report shows that people who work outside the home got COVID at about the average rate for the state.
 
+# Vaccines Q&A
+
+## How should we think about people who have been vaccinated?
+
+There are two questions that are important for understanding vaccines:
+
+1. How much safer for the vaccinated individual is exposure to COVID?
+2. How much less likely is it for a vaccinated individuals to expose others to COVID? (Or equivalently, how much less likely is a vaccinated individual to catch COVID?)
+
+We address these two questions individually.
+
+### 1. How much safer for the vaccinated individual is exposure to COVID?
+All currently approved vaccines are proven significantly reduce the chance of getting sick. If you do get sick after being vaccinated, you are significantly less likely to be hospitalized or die. However, one of the scary things about COVID is that cases with no outward symptoms have been found to cause severe lung damage [source?]. As a result:
+
+- For people who are trying to avoid COVID to avoid long term health problems, we do not recommend changing your budget (beyond what the reduction in microCOVIDs from #2 allows you to do).
+- For people who are trying to avoid COVID to avoid hospitalization or death as a direct result of COVID (generally those who are more vulnerable to COVID), having both doses of the AstraZeneca, Pfizer, or Moderna vaccines and waiting 1-2 weeks after the second dose should reduce your chance of death by an additional 2-10x over the microCOVID reduction outlined in part 2. As such, you may consider increasing your microCOVID budget from 0.1% / year to somewhere between 0.2% / year and 1% / year.
```

Comment:
First - good work on the writeup! I am really confused by this section and the recommendations in it - sharing feedback as one user data point :)

1. The Pfizer/Moderna vaccines reduce the risk of death by more than 10x, right? (My understanding is that they're ~95% effective against all symptomatic disease and more effective against severe disease). Is the "2-10x" intended to avoid "double counting" risk reductions with the microCOVID reductions below?

2. I'm confused about the relationship between this section and the one below it. This section says it's about "how much safer is the vaccinated person", but it then tells the reader to use the reductions listed in section 2, which is about transmission. I'm having trouble understanding which multipliers are intended to apply to which situations - it sounds like maybe the microCOVID reductions in section 2 really apply to the first question also?

If it helps, my core questions are "how safe am I if I'm vaccinated", "how much safer are the people I come into contact with post-vaccination", and "how much more risk can I responsibly take on post-vaccination without increasing my pod's risk"

Reply:
Hi cstigler, thanks for dropping in on the review!

I've done some heavy re-writes based on your + Jeremy's reviews. ptal.

I also had to dig more heavily into the data to check what I was saying:

1. Actually... no? Pfizer/Moderna vaccinated group had 0 deaths (which people are doing a lot of chest-thumping about), but the control groups only had 34 cases of severe covid and 1 death. That's actually not enough data points to make meaningful claims. The 2-10x was intended to avoid "double counting" when I thought the vaccines were ~98% effective at preventing severe covid (0.2x chance of getting covid -> 0.2x as many microCOVIDs. 0.02x chance of severe infection / 0.2x as many microCOVIDs = 0.1x severity per microCOVID), but having dug deeper I can't substantiate this efficacy.
2. Yes, the microCOVID reductions apply to the first. I've re-ordered the questions to make this flow better.

> "how safe am I if I'm vaccinated"

0.2x multiplier on all activities. Additionally, each microCOVID is maybe 2-4x less likely to land you in the hospital. Maybe.

> "how much safer are the people I come into contact with post-vaccination"

I'm not quite sure how to interpret this question. If you mean "how much safer are the people I come into contact with who have been vaccinated", then the answer is they get a 0.2x multiplier on all of their activities.

> "how much more risk can I responsibly take on post-vaccination without increasing my pod's risk"

Again, ambiguous question.

* If only you have ben vaccinated, then you cannot change how many microCOVIDs you bring home (but you get the 0.2x multiplier on your activities, so you can do activities that would be 5x more risky to an unvaccinated person). Your housemates can take home 5x more risk without increasing your risk past what it was when you were unvaccinated.
* If your entire pod has been vaccinated, you all can bring home 5x more microCOVIDs because you will get a 0.2x multiplier from each other. I don't think you should do that because community spread is still an issue, and you generally can't get microCOVIDs without giving some to someone else as well.

Reply:
Just gave it a read-through, the updated version makes a lot of sense to me and looks good! Also this line in your response really resonated / was easy to follow for me: "0.2x multiplier on all activities. Additionally, each microCOVID is maybe 2-4x less likely to land you in the hospital. Maybe." I think that's basically the one-line answer to what I was trying to ask: how much less should I worry about transmitting to other people, and how much less should I worry about my own health?

Excited to see this rolling out!

Reply:
Great. I added that as a TL;DR.